py/mpconfig.h

```diff
@@ -1311,6 +1311,11 @@ typedef double mp_float_t;
 #define MICROPY_PY_CMATH (MICROPY_CONFIG_ROM_LEVEL_AT_LEAST_EXTRA_FEATURES)
 #endif
 
+// Whether to provide "micropython" module
+#ifndef MICROPY_PY_MICROPYTHON
+#define MICROPY_PY_MICROPYTHON (1)
```

Comment:
I think this should be `MICROPY_CONFIG_ROM_LEVEL_AT_LEAST_CORE_FEATURES`

Reply:
Done. Since this has a bit more impact (and causes a test to break), I did it in a follow-up commit.

I tweaked the `nanbox_smallint` test to skip if `micropython` cannot be imported.